### DISCUSSION TITLE
How to access GUI of Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by Rutvij-Brahmbhatt** [microsoft/WSL#6598] [Q&A] [answered]
Hi Guys,

I have installed Ubuntu 20.04 using WSL2. I can access the terminal fine. However, wondering how to access the GUI?? Is this even supported??

Kind Rgds,
Rutvij

**Comment by craigloewen-msft** [comment, marked as answer]
We don't yet support GUI apps out of the box, but are working on adding support for it in the future! :) In the meantime you can install a 3rd party x server and use it, like [in this video](https://www.youtube.com/watch?v=hhxMFVQsklI&feature=emb_title).

**Reply by Rutvij-Brahmbhatt** [reply]
Thanks Craig - Much appreciated!!!

Love the work you guys doing on WSL as I always wanted the benefits of both worlds in one :)

**Comment by Biswa96** [comment]
Maybe coming soon https://www.youtube.com/watch?v=wK4JPTxcGOE
